Observing-Interface-Setup/environments.rst

```diff
@@ -0,0 +1,174 @@
+
+.. _Observing-Interface-Environments-NTS:
+
+NCSA Test Stand (NTS)
+---------------------
+
+The NTS is currently being used as a staging platform to test software deployment prior to deployment at the summit.
+Systems that control hardware at the :ref:`Summit <Observing-Interface-Environments-Summit>` run here in simulation mode.
+Once deployment is tested here, the platform is made available for general users for training, testing and developing of control algorithm and procedures.
+
+- Nublado: https://lsst-nts-k8s.ncsa.illinois.edu/
+- Chronograf: https://lsst-chronograf-nts-efd.ncsa.illinois.edu/
+- LOVE:
+    .. prompt:: bash
+
+       ssh -L 8080:lsst-teststand-ts1.ncsa.illinois.edu:80 -J lsst-login03.ncsa.illinois.edu lsst-teststand-ts1.ncsa.illinois.edu
+
+    - Use your NCSA account to login when prompted.
+    - You will be prompted for your password twice after running the command above.
+    - Once logged in, open http://localhost:8080/ on a browser.
+
+- ArgoCD: https://lsst-argocd-nts-efd.ncsa.illinois.edu/
+- Slack: ncsa-integ-teststand
+
+
+.. _Observing-Interface-Environments-BTS:
+
+Base Test Stand (BTS)
+---------------------
+
+BTS is a simulation environment running at the base facility in La Serena.
+The main difference between the :ref:`NTS <Observing-Interface-Environments-NTS>` and this environment is that some systems will run using hardware simulators.
+
+Hardware simulators are, in general, higher fidelity than those of pure software available at :ref:`NTS <Observing-Interface-Environments-NTS>`.
+For that, BTS will be used mostly for higher fidelity testing, development and debugging.
+
+- Nublado: https://base-lsp.lsst.codes/
+- Chronograf: https://chronograf-base-efd.lsst.codes/
+- LOVE: N/A
+- ArgoCD: https://base-lsp.lsst.codes/argo-cd/
+- Slack: N/A
+
+.. _Observing-Interface-Environments-TTS:
+
+Tucson Test Stand (TTS)
+-----------------------
+
+TTS is being assembled in Tucson and is still not available for use.
+This environment will be similar to that of :ref:`NTS <Observing-Interface-Environments-NTS>` and will be used mainly for development of deployment strategies.
+
+- Nublado: https://tucson-teststand.lsst.codes/
+- Chronograf: https://chronograf-tucson-teststand-efd.lsst.codes/
+- LOVE: N/A
+- ArgoCD: https://tucson-teststand.lsst.codes/argo-cd
+- Slack: N/A
+
+.. _Observing-Interface-Environments-Summit:
+
+Summit
+------
+
+Summit is our main production environment.
+Systems running here will be directly controlling hardware or communicating with components that control actual hardware.
+
+  .. important::
+
+      In the case of the Summit it is required to have personnel present at the site prior to any activity that involves moving hardware.
+      You may also be required to issue a jira ticket in the `summit activity project <https://jira.lsstcorp.org/projects/SUMMIT>`__.
```

Comment:
So not just anyone is allowed to file tickets here at the moment (permissions wise you can, procedure wise you can't)... 
Maybe just say it has to be a planned activity and put into the "to-do" state by the personnel organizing summit activities?